Source/Core/DolphinWX/Input/HotkeyInputConfigDiag.cpp

```diff
@@ -68,6 +69,31 @@ HotkeyInputConfigDialog::HotkeyInputConfigDialog(wxWindow* const parent, InputCo
   // Frame advance is an example of a typical TAS tool.
   notebook->AddPage(tab_tas, _("TAS Tools"));
 
+  if (withDebugging)
+  {
+    // Debugging
+    auto* const tab_debugging = new wxPanel(notebook);
```

Comment:
This layout creation should likely be in its own function that either returns the new wxPanel instance, or the sizer that goes into the panel (rule of thumb: if you have to leave a comment explaining *what* something is, rather than *why* you're doing something, then it's probably best suited for its own function with a descriptive name).